components/eventing-controller/testing/subscriber.go

```diff
@@ -0,0 +1,103 @@
+package testing
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/avast/retry-go"
+	pkgerrors "github.com/pkg/errors"
+)
+
+type Subscriber struct {
+	addr   string
+	t      *testing.T
+	server *http.Server
+}
+
+func NewSubscriber(addr string, t *testing.T) *Subscriber {
+	return &Subscriber{
+		addr: addr,
+		t:    t,
+	}
+}
+
+func (s *Subscriber) Start() {
+	store := ""
+	mux := http.NewServeMux()
+	mux.HandleFunc("/subscribe", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is it better to rename this endpoint to `/store` to reflect what's it really doing?

Reply:
Changed to `/store`